### ISSUE TITLE
UI/UX: Hide compose FAB when About tab / editing

### ISSUE THREAD

**Issue by sk22** [mastodon/mastodon-android#183]
![Screenshot_20220522-030649807](https://user-images.githubusercontent.com/6217438/169674033-776f22c6-3193-4065-af92-56bdd14f633c.jpg)

I think that it can be misleading to have the compose button still there while in the About tab, or even editing one's profile. Since it's an edit icon, it could also refer to editing (or saving, maybe) the profile. (I actually pressed the button accidentally at least once for this reason)

**Comment by realpixelcode** [comment]
Yes, it shouldn't be possible in the first place to compose a toot while editing your profile.
